kodi_addon_checker/check_dependencies.py

```diff
@@ -32,6 +35,12 @@ def check_addon_dependencies(report: Report, repo_addons: dict, parsed_xml, bran
             if required_addon not in ignore:
                 report.add(Record(
                     PROBLEM, "Required addon %s not available in current repository." % required_addon))
+            else:
+                version = VERSION_ATTRB[required_addon][branch_name]
```

Comment:
this can reference unknown keys, have you tested what happens then? Do we just crash?

Reply:
Fixed